Examples/src/components/Search/Search.tsx

```diff
@@ -9,15 +9,17 @@ import { makeStyles } from "@material-ui/core/styles";
 
 const useStyles = makeStyles(theme => ({
     root: {
-        width: "100%",
+        display: "none",
         [theme.breakpoints.up("sm")]: {
-            width: "80ch"
+            width: "100%",
```

Comment:
Could you remove it to scss file, please

Reply:
Extracted all other material-ui styles for this component into a stylesheet